Resources/Locale/en-US/objectives/conditions/steal-target-groups.ftl

```diff
@@ -0,0 +1,66 @@
+steal-target-groups-stamps = stamps (any)
+steal-target-groups-door-remotes = door remotes (any)
+steal-target-groups-encryption-keys = encryption keys (any)
+steal-target-groups-id-cards = ID Cards (any)
+steal-target-groups-lamps = LAMPS
+
+# Thief single item
+steal-target-groups-forensic-scanner = {$ ent-ForensicScanner.name }
+steal-target-groups-flippo-engraved-lighter = {$ ent-FlippoEngravedLighter.name }
+steal-target-groups-ammo-tech-fab-circuitboard = {$ ent-AmmoTechFabCircuitboard.name }
+steal-target-groups-clothing-head-hat-warden = {$ ent-ClothingHeadHatWarden.name }
+steal-target-groups-clothing-outer-hardsuit-void-paramed = {$ ent-ClothingOuterHardsuitVoidParamed.name }
+steal-target-groups-medical-tech-fab-circuitboard = {$ ent-MedicalTechFabCircuitboard.name }
+steal-target-groups-clothing-headset-alt-medical = {$ ent-ClothingHeadsetAltMedical.name }
+steal-target-groups-research-and-development-server-machine-circuitboard = {$ ent-ResearchAndDevelopmentServerMachineCircuitboard.name }
+steal-target-groups-fire-axe = {$ ent-FireAxe.name }
+steal-target-groups-ame-part-flatpack = {$ ent-AmePartFlatpack.name }
+steal-target-groups-salvage-expeditions-computer-circuitboard = {$ ent-SalvageExpeditionsComputerCircuitboard.name }
+steal-target-groups-cargo-shuttle-console-circuitboard = {$ ent-CargoShuttleConsoleCircuitboard.name }
+steal-target-groups-salvage-shuttle-console-circuitboard = {$ ent-SalvageShuttleConsoleCircuitboard.name }
+steal-target-groups-clothing-eyes-hud-beer = {$ ent-ClothingEyesHudBeer.name }
+steal-target-groups-bible = {$ ent-Bible.name }
+steal-target-groups-clothing-neck-goldmedal = {$ ent-ClothingNeckGoldmedal.name }
+steal-target-groups-clothing-neck-clownmedal = {$ ent-ClothingNeckClownmedal.name }
+
+# Thief structures
+steal-target-groups-teg = teg generator part
+steal-target-groups-freezer-heater = freezer or heater
+steal-target-groups-altar-nanotrasen = nanotrasen altar (any)
+
+steal-target-groups-nuclear-bomb = {$ ent-NuclearBomb.name }
+steal-target-groups-fax-machine-captain = {$ ent-FaxMachineCaptain.name }
+steal-target-groups-chem-dispenser = {$ ent-ChemDispenser.name }
+steal-target-groups-xeno-artifact = {$ ent-BaseXenoArtifact.name }
+steal-target-groups-booze-dispenser = {$ ent-BoozeDispenser.name }
+steal-target-groups-plant-rd = {$ ent-PottedPlantRD.name }
+steal-target-groups-toilet-golden-dirty-water = {$ ent-ToiletGoldenDirtyWater.name }
+
+# Thief Animal
+steal-target-groups-animal-named-cat = CMO's Cat
+
+steal-target-groups-animal-ian = {$ ent-MobCorgiIan.name }
+steal-target-groups-animal-mc-griff = {$ ent-MobMcGriff.name }
+steal-target-groups-animal-walter = {$ ent-MobWalter.name }
+steal-target-groups-animal-morty = {$ ent-MobPossumMorty.name }
+steal-target-groups-animal-renault = {$ ent-MobFoxRenault.name }
+steal-target-groups-animal-shiva = {$ ent-MobSpiderShiva.name }
+steal-target-groups-animal-tropico = {$ ent-MobCrabAtmos.name }
```

Comment:
The localization file will shrink to the following form. 

```suggestion
steal-target-groups-unknown = unknown item

# Thief Collection
steal-target-groups-figurines = figurines (any)
steal-target-groups-heads-cloaks = head's cloaks (any)
steal-target-groups-heads-bedsheets = head's bedsheets (any)
steal-target-groups-stamps = stamps (any)
steal-target-groups-door-remotes = door remotes (any)
steal-target-groups-encryption-keys = encryption keys (any)
steal-target-groups-id-cards = ID Cards (any)
steal-target-groups-lamps = LAMPS
```

But I also ask that you consider redoing the declension of the name. After all, if you drop a target on a collection, you get an entry of the form `encryption keys (any)s`. This is because the system uses the key where the `MAKEPLURAL` function is called. Therefore, either remove this function or change the localization strings for the targets.
https://github.com/space-wizards/space-station-14/blob/4da704fd9c15df803980cedbca992048abc51343/Resources/Locale/en-US/objectives/conditions/steal.ftl#L11